config/config_cyto.yaml

```diff
@@ -0,0 +1,113 @@
+ # Global workflow control
```

Comment:
Let's add the Cytoscape step to `config/config.yaml` and remove this additional config file.

Reply:
done